site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -110,3 +115,30 @@ curl -GET "http://localhost:9358/gameserver" -H "accept: application/json"
 ```
 {"object_meta":{"creation_timestamp":"-62135596800","labels":{"agones.dev/sdk-foo":"bar"}},"spec":{"health":{}},"status":{"state":"Ready"}}
 ```
+
+## Running Local Mode in a Container
+
+Once you have your game server process in a container, you may also want to test the container build locally as well.
+
+Since the production agones-sdk binary has the `--local` mode built in, you can also use the production conatiner image
+locally as well!
+
+Since the SDK and your game server container need to share a port of `localhost`, one of the easiest ways to do that 
+is to have them both run with their networking occurring on the host, like so:
```

Comment:
I would find it clearer to say "run using the host network, like so"

Reply:
I like it 👍🏻